Makefile

```
@@ -21,6 +21,9 @@ check:
 test:
 	cargo test --profile optimized-dev --all-targets --all-features
 
+proptests:
```

I think maybe keeping it in singular is more in line with the rest of the rules (ex. `test` vs `tests`).
```suggestion
proptest:
```